playbooks/openstack/openshift-cluster/provision.yml

```diff
@@ -48,13 +48,22 @@
   become: yes
   gather_facts: yes
   roles:
```

Comment:
If no roles are defined in this play, let's remove this line.

Reply:
Gotcha.